package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lingui": "lingui",
-    "django:makemessages": "python manage.py makemessages -e py,html -i coverage --no-wrap",
+    "django:makemessages": "python manage.py makemessages --all -e py,html -i coverage --no-wrap",
```

Comment:
I think that at some point, Django's `makemessages` command changed from _defaulting_ to processing all locales to requiring that `--all` be passeed-in to explicitly tell it to process all locales.  This must have happened during one of our recent upgrades of Django but it wasn't caught because this command isn't run during CI or anything.